src/lib/Libattr/master_resv_attr_def.xml

```diff
@@ -780,6 +780,23 @@
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
    </attributes>
+   <attributes>
+   /* RESV_ATR_del_idle_time */
+	<member_name><both>ATTR_del_idle_time</both></member_name>	<!-- delete_idle_time -->
+	<member_at_decode>decode_time</member_at_decode>
+	<member_at_encode>encode_time</member_at_encode>
+	<member_at_set>set_l</member_at_set>
+	<member_at_comp>comp_l</member_at_comp>
+	<member_at_free>free_null</member_at_free>
+	<member_at_action>NULL_FUNC</member_at_action>
+	<member_at_flags><both>READ_WRITE</both></member_at_flags>
+	<member_at_type><both>ATR_TYPE_LONG</both></member_at_type>
+	<member_at_parent>PARENT_TYPE_RESV</member_at_parent>
+	<member_verify_function>
+	<ECL>NULL_VERIFY_DATATYPE_FUNC</ECL>
```

Comment:
Should there be a verify function? Maybe verify_datatype_long?

Reply:
We don't need one.  It should only be set by the server.  The verify functions are client side.